### ISSUE TITLE
Is it possible to run your tests with ava 4.3.1 ?

### ISSUE THREAD

**Issue by Tas-sos** [elycruz/rollup-plugin-sass#97]
Hello, I am trying to update the debian package to your latest version, but I cannot.

* https://tracker.debian.org/pkg/node-rollup-plugin-sass
* https://salsa.debian.org/js-team/node-rollup-plugin-sass

```
apt info ava
Package: ava
Version: 4.3.1+dfsg+~cs43.2.10-1
Priority: optional
Section: javascript
Source: node-ava
Maintainer: Debian Javascript Maintainers <pkg-javascript-devel@lists.alioth.debian.org>
Installed-Size: 601 kB
Provides: node-arrgv (= 1.0.2), node-ava (= 4.3.1), node-ava-typescript (= 3.0.1), node-chunkd (= 2.0.1), node-ci-parallel-vars (= 1.0.1), node-code-excerpt (= 4.0.0), node-common-path-prefix (= 3.0.0), node-convert-to-spaces (= 2.0.1), node-is-error (= 2.2.2), node-p-event (= 5.0.1), node-pkg-conf (= 4.0.0), node-serialize-error (= 11.0.0), node-supertap (= 3.0.1), node-temp-dir (= 2.0.0)
Depends: node-acorn, node-ansi-styles, node-arrify, node-cbor, node-chalk (>= 5), node-chokidar, node-ci-info, node-clean-yaml-object, node-cli-truncate, node-cliui (>= 7.0.4+repack+~cs3.1.0-3~), node-concordance, node-currently-unhandled, node-debug, node-del, node-emittery, node-escape-string-regexp, node-execa, node-find-up (>= 6), node-globby, node-ignore-by-default, node-indent-string, node-inquirer, node-is-plain-object, node-is-promise, node-js-yaml, node-load-json-file (>= 7), node-matcher, node-mem, node-ms, node-p-map, node-p-timeout, node-picomatch, node-plur, node-pretty-ms, node-read-pkg, node-resolve-cwd, node-resolve-from, node-slash, node-stack-utils, node-strip-ansi, node-write-file-atomic, node-yargs, nodejs:any (>= 14)
Homepage: https://ava.li
Download-Size: 137 kB
APT-Manual-Installed: yes
APT-Sources: http://deb.debian.org/debian sid/main amd64 Packages
Description: Futuristic test runner 🚀
 Even though JavaScript is single-threaded, IO in Node.js can happen in
 parallel due to its async nature. AVA takes advantage of this and runs your
 tests concurrently, which is especially beneficial for IO heavy tests. In
 addition, test files are run in parallel as separate processes, giving you
 even better performance and an isolated environment for each test file.
 .
 Switching from Mocha to AVA in Pageres brought the test time down from 31 to
 11 seconds. Having tests run concurrently forces you to write atomic tests,
 meaning tests don't depend on global state or the state of other tests, which
 is a great thing!
 .
 Node.js is an event-based server-side JavaScript engine.
 ```
 
 ```
apt info ts-node
Package: ts-node
Version: 10.8.2+~cs8.8.29-1
Priority: optional
Section: javascript
Maintainer: Debian Javascript Maintainers <pkg-javascript-devel@lists.alioth.debian.org>
Installed-Size: 942 kB
Provides: node-cspotcode-source-map-support (= 0.8.1), node-ts-node (= 10.8.2), node-tsconfig-node10 (= 1.0.9), node-tsconfig-node12 (= 1.0.11), node-tsconfig-node14 (= 1.0.3), node-tsconfig-node16 (= 1.0.3), node-tsconfig-node18 (= 1.0.1), node-v8-compile-cache-lib (= 3.0.1)
Depends: node-acorn, node-ampproject-remapping, node-arg, node-builtin-modules, node-create-require, node-diff, node-make-error, node-typescript, node-yn, nodejs:any
Homepage: https://github.com/TypeStrong/ts-node
Download-Size: 167 kB
APT-Manual-Installed: yes
APT-Sources: http://deb.debian.org/debian sid/main amd64 Packages
Description: TypeScript execution and language shell
 Provides a command-line interface to execute TypeScript code
 as a shell and an interactive read-eval-print-loop (REPL).
 ```

I have the following problems with tests :
```
+ ava ./test/index.test.ts ./test/style.test.ts -s

  
  ✖ Timed out while running tests

  
  ✖ Timed out while running tests

  ─
  ```

or 
```
ava ./test/index.test.ts ./test/style.test.ts --serial --verbose -T 2m

  ✖ test/index.test.ts exited due to an error:

  Error [ERR_WORKER_OUT_OF_MEMORY]: Worker terminated due to reaching memory limit: JS heap out of memory

  Error [ERR_WORKER_OUT_OF_MEMORY]: Worker terminated due to reaching memory limit: JS heap out of memory
      at new NodeError (node:internal/errors:372:5)
      at Worker.[kOnExit] (node:internal/worker:276:26)
      at Worker.<computed>.onexit (node:internal/worker:198:20)

  Uncaught exception in test/style.test.ts

  test/style.test.ts(1,15): error TS2724: '"ava"' has no exported member named 'before'. Did you mean 'BeforeFn'?
  test/style.test.ts(17,3): error TS2322: Type 'DOMWindow' is not assignable to type 'Window & typeof globalThis'.
    Type 'DOMWindow' is not assignable to type 'Window'.
      Types of property 'self' are incompatible.
        Type 'DOMWindow' is not assignable to type 'Window & typeof globalThis'.
          Type 'DOMWindow' is missing the following properties from type 'typeof globalThis': AnalyserNode, Animation, AnimationEffect, AnimationEvent, and 500 more.
  

  ✖ test/style.test.ts exited with a non-zero exit code: 1
  ─

  1 uncaught exception
```

Thank you in advance.

**Comment by elycruz** [comment]
@Tas-sos Hi, tracking issue for related update created and assigned - issue #98.

**Comment by elycruz** [comment]
@Tas-sos Lol, ignore comment edit ( 👍🏽 ).

**Comment by elycruz** [comment]
Hey @Tas-sos , issue addressed, and published in [release v1.12.13](https://github.com/differui/rollup-plugin-sass/releases/tag/1.12.13) (fyi).
